build.gradle

```diff
@@ -233,9 +233,9 @@ dependencies {
   compile 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.9.7'
 
   // logging
-  compile 'org.apache.logging.log4j:log4j-api:2.11.0'
-  compile 'org.apache.logging.log4j:log4j-core:2.11.0'
-  compile 'org.apache.logging.log4j:log4j-1.2-api:2.11.0'
+  compile 'org.apache.logging.log4j:log4j-api:2.11.2'
```

Comment:
If we are using slf4j, wouldn't it be the case to replace log4j-api with slf4j api?

Reply:
Ah nvm...